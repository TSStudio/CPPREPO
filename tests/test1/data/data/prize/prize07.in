7
a 78 79 Y
b 89 90 N
c 78 87 Y
d 88 77 N
e 86 85 Y
f 88 78 N
g 77 88 Y