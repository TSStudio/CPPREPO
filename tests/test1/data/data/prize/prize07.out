e
10000
22000